src/Nox.Types.Abstractions/Enums/File/FileFormatType.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Enumeration for file format types.
+/// </summary>
+public enum FileFormatType
+{
+    /// <summary>
+    /// Represents all file format types.
+    /// </summary>
+    All,
```

Comment:
to have this the enum must be decorated with Flags, and the All must be All = pdf | ....